src/pages/attestering/Attestering.tsx

```diff
@@ -0,0 +1,242 @@
+import yup, { formikErrorsTilFeiloppsummering, formikErrorsHarFeil } from '~lib/validering';
+import VisBeregning from '~pages/saksoversikt/beregning/VisBeregning';
+import { Simulering } from '~pages/saksoversikt/simulering/simulering';
+import { useAppSelector, useAppDispatch } from '~redux/Store';
+
+import messages from './attestering-nb';
+import styles from './attestering.module.less';
+
+type Props = {
+    sak: Sak;
+    behandlingId: string;
+};
+
+const VilkårsvurderingInfoLinje = (props: { type: Vilkårtype; verdi: Vilkårsvurdering }) => (
+    <div className={styles.infolinjeContainer}>
+        <div className={styles.infolinje}>
+            <span className={styles.statusContainer}>
+                <span className={styles.statusIcon}>{statusIcon(props.verdi.status)}</span>
+            </span>
+            <div>
+                <span className={styles.infotittel}>{vilkårTittelFormatted(props.type)}:</span>
+                <p>{props.verdi.begrunnelse ? props.verdi.begrunnelse : ''}</p>
+            </div>
+        </div>
+    </div>
+);
+
+const VilkårsOppsummering = (props: { behandling: Behandling; sakId: string }) => {
+    return (
+        <div>
+            <Innholdstittel className={styles.tittel}>Vilkårsvurderinger</Innholdstittel>
+            {Object.entries(props.behandling.vilkårsvurderinger).map(([k, v]) => (
+                <VilkårsvurderingInfoLinje type={k as Vilkårtype} verdi={v} key={k} />
+            ))}
+        </div>
+    );
+};
+
+const VisDersomSimulert = (props: { sak: Sak; behandling: Behandling }) => {
+    if (props.behandling.status === Behandlingsstatus.AVSLÅTT) {
+        return null;
+    }
+    if (props.behandling.beregning) {
+        return (
+            <>
+                <VisBeregning beregning={props.behandling.beregning} />
+                <div>
+                    <Innholdstittel>Oppdragssimulering</Innholdstittel>
+                    <Simulering sak={props.sak} behandlingId={props.behandling.id} />
+                </div>
+            </>
+        );
+    }
+    return <>Det er ikke gjort en beregning</>;
+};
+enum Grunn {
+    FEIL_I_BEREGNING = 'FEIL_I_BEREGNING',
+    FEIL_I_PERIODE = 'FEIL_I_PERIODE',
+}
+interface FormData {
+    beslutning?: boolean;
+    grunn?: string;
+    begrunnelse?: string;
+}
+const Attestering = (props: Props) => {
+    const { sak, behandlingId } = props;
+    const [hasSubmitted, setHasSubmitted] = useState(false);
+    const attesteringStatus = useAppSelector((s) => s.sak.attesteringStatus);
+    const intl = useI18n({ messages });
+    const dispatch = useAppDispatch();
+
+    const behandling = sak.behandlinger.find((x) => x.id === behandlingId);
+    if (!behandling) {
+        return <AlertStripe type="feil">Fant ikke behandlingsid</AlertStripe>;
+    }
+    const formik = useFormik<FormData>({
+        initialValues: {},
+        onSubmit: (values) => {
+            const { beslutning } = values;
+
+            if (beslutning) {
```

Comment:
Nitpick: hvorfor destukturere `values` og hele sulamitten i stedet for bare
```suggestion
            if (values.beslutning) {
```
?